app/webpage/main.js

```diff
@@ -0,0 +1,97 @@
+$(document).ready(function () {
+  //object for easy manipulation of title, description, and day of week of task
+  var task = {
+    title: '',
+    description: '',
+    weekday: 0
+  }
+  
+  //initiation of the webpage's first screen
+  $("#password").hide();
+  $("#sign-up").hide();
+  $("#option-help").hide();
+  $("#week-scheduler").hide();
+  $("#email").val("");
+  $("#pswd").val("");
+  
+/* lgin-user-01, plan-head-01 */
+  $("#log-email").click(eValidate);
+  
+/* lgin-pswd-01 */
+  $("#log-pswd").click(pswdValidate);
+  
+/* lgin-npwd-01 */
+  $("#log-create").click(signupValidate);
+  
+/* plan-head-02-03 */
+  $("#help").click(helpText);
+  
+  $(".wkday").click(function() { setDay($(this).text(), task); });
+});
+
+/* lgin-user-01 */
+function eValidate() {
+  if($("#email").val() == "" || $("#email").val() == undefined){
```

Comment:
Also, you should save the value to a variable if you are going to access it twice, because otherwise jQuery searches the DOM both times.
```
var current_email = $("#email").val();
if(current_email...
```